### PR TITLE
fix: hue values over 255 are cut off

### PR DESCRIPTION
**Pull request by rpetersen27**
There is some code, that min/max the parsed values to 0 and 255. For hsl input, that is not true as the first value should be within 0 and 360 and the second and third within 0 and 100. This fix reuses the min/max ranges from the color-space library. 

Additionally, I rounded the result of the hsl-function, as rbg just support shorts anyways. But I am not sure, if there are reasons, why that should be kept as floating point numbers. I would really look forward to some feedback on this.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,13 +17,16 @@ module.exports = function rgba (color) {
...
-	values[0] = Math.min(Math.max(parsed.values[0], 0), 255)
-	values[1] = Math.min(Math.max(parsed.values[1], 0), 255)
-	values[2] = Math.min(Math.max(parsed.values[2], 0), 255)
+	values[0] = Math.min(Math.max(parsed.values[0], minLimits[0]), maxLimits[0])
+	values[1] = Math.min(Math.max(parsed.values[1], minLimits[1]), maxLimits[1])
+	values[2] = Math.min(Math.max(parsed.values[2], minLimits[2]), maxLimits[2])
 
 	if (parsed.space[0] === 'h') {
-		values = hsl.rgb(values)
+		values = hsl.rgb(values).map(Math.round)
```
Comment (dy) [inline index.js:29]: why is there Math.round?

**Comment by dy** [comment]
Good catch!

**Comment by dy** [comment]
Not sure why round though. CSS supports [floating-point](https://developer.mozilla.org/en-US/docs/Web/CSS/number) channel values, eg `rgb(10.8, 28, 36.7)` . The definition is `rgb[a](R, G, B[, A])`.
Would be weird to force-round for user, if initial value is more precise. There're multiple cases when precise value may be needed, eg. intermediary value between space conversions, calculated/css-calc value, and so on. Not sure what service losing precision does for consumer, also that's breaking change.

**Comment by dy** [comment]
`rgb(0,0,0)` what da heck?

**Comment by rpetersen27** [comment]
Yeah, main reason was unit tests. results in some long floating numbers. 
I will remove the Math.round with the next commit.

**Comment by rpetersen27** [comment]
Oh man, I totally forgot this PR, I am sorry. I added an additional commit that removes the "Math.round" und adjusted the tests accordingly.

**Comment by dy** [comment]
Thanks. LGTM. A little ask. Can you rename `maxLimits` to just `max`? (personal preference to keep even unminified code short, no need for redundant verbosity)

**Comment by rpetersen27** [comment]
I changed `minLimits` as well.

**Pull request merged**